charts/flyte-core/templates/clusterresourcesync/deployment.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: syncresources
+  name: {{ template "flyteclusterresourcesync.name" . }}
```

Comment:
This will technically recreate the deployment when it renames, not sure if that is an issue.

Reply:
does this re-create the deployment?  or does this create a new deployment?

Reply:
new, it will tear down the old deployment and make a new one. it renames this to `flyteclusterresourcesync`

https://github.com/flyteorg/flyte/blob/master/charts/flyte-core/templates/_helpers.tpl#L62